doc/api/net.md

```diff
@@ -2,9 +2,14 @@
 
 > Stability: 2 - Stable
 
-The `net` module provides you with an asynchronous network wrapper. It contains
-functions for creating both servers and clients (called streams). You can include
-this module with `require('net');`.
+The `net` module provides an asynchronous network API for creating stream-based
+servers ([`net.Server`][]) and clients ([`net.Socket`][]) that implement TCP
```

Comment:
The argument for the `'connection'` listener is a `net.Socket`, that’s where the non-client sockets come from.

> I guess saying it provides clients would help people looking for one.

Not disagreeing! It’s definitely good to state that `net` provides servers and clients… I’m just trying to say that I would prefer to avoid any confusion of associating `net.Socket` with `client`.

If you think this is a non-issue, feel free to ignore this :)